data/members/clementd.yml

```diff
@@ -0,0 +1,16 @@
+id: clementd
+display_name: Clément Delafargue
+bio: Functional programmer, baker of biscuits
+avatar: "clementd.jpg"
+location: France, Nantes
+main_link:
+  url: https://clementd.wtf
+  lang: fra
```

Comment:
Hi, I sligthly disagree with `fra` ;) you can just remove the field (here and in your main_feed)

Reply:
right